RUFAS/weather.py

```diff
@@ -16,33 +15,23 @@ class Weather:
     """
     The `Weather` class manages all weather data used to run a single simulation.
 
+    Parameters
+    ----------
+    weather_file : dict[str, List[Any]]
+        The weather dictionary read from the provided weather file.
```

Comment:
I think make this more general since it won't necessarily always be a file we're pulling weather from.

```suggestion
        The weather dictionary read from the provided weather input source.
```